### DISCUSSION TITLE
crystal orientation/directions

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#28211] [Q&A Modules: Solid mechanics]
Hello, 

I am using the update method and bicrystal crystal plasticity model. Before I apply any Euler angles, what are the original or automatic crystal directions for the crystals in each model? I am assuming it is x : [100] , y : [010] , z : [001].

**Comment by GiudGiud** [comment]
@laagesen

**Reply by laagesen** [reply]
I think @vprithiv  can answer this better

**Reply by sapitts** [reply]
The assumption is correct: with out any user-supplied Euler-angles, the crystal directions are aligned with the Cartesian axes
